### PR TITLE
qa_crowbarsetup: use SES 2.1 for cloud6

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

**Comment by vuntz** [comment]
There are a couple of other lines that need an update: 527 and 1176.

Also wondering if it should be an option until we have completed the switch to 2.1?

**Comment by toabctl** [comment]
FYI: there is https://trello.com/c/Z51o9UGe/679-3-s32p4-move-to-ses-2-1 for this topic

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1173,13 +1171,13 @@ function create_repos_yml()
...
         SLES12-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12:/x86_64/update/ \
         SLE12-HA-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12:/x86_64/update/ \
-        SUSE-Enterprise-Storage-2-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/Storage:/2:/x86_64/update/ \
         >> $tmp_yml
 
     create_repos_yml_for_platform "suse-12.1" "x86_64" "$tftpboot_repos12sp1_dir" \
         SLES12-SP1-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12-SP1:/x86_64/update/ \
         SLE12-SP1-HA-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12-SP1:/x86_64/update/ \
         SUSE-OpenStack-Cloud-6-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/OpenStack-Cloud:/6:/x86_64/update/ \
+        SUSE-Enterprise-Storage-2.1-Updates-test=http://dist.suse.de/ibs/SUSE:/SLE-12-SP1:/Update:/Products:/SES2.1/standard/ \
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: I think this is not an updates-test repo, but the normal one for released updates.

Comment (mjura) [inline scripts/qa_crowbarsetup.sh]: I've changed repo name for it

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -1173,13 +1171,13 @@ function create_repos_yml()
...
         SLES12-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12:/x86_64/update/ \
         SLE12-HA-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12:/x86_64/update/ \
-        SUSE-Enterprise-Storage-2-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/Storage:/2:/x86_64/update/ \
         >> $tmp_yml
 
     create_repos_yml_for_platform "suse-12.1" "x86_64" "$tftpboot_repos12sp1_dir" \
         SLES12-SP1-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12-SP1:/x86_64/update/ \
         SLE12-SP1-HA-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12-SP1:/x86_64/update/ \
         SUSE-OpenStack-Cloud-6-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/OpenStack-Cloud:/6:/x86_64/update/ \
+        SUSE-Enterprise-Storage-2.1-Product-test=http://dist.suse.de/ibs/SUSE:/SLE-12-SP1:/Update:/Products:/SES2.1/standard/ \
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: URL should still be something like http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/Storage:/2.1:/x86_64/update/

Comment (mjura) [inline scripts/qa_crowbarsetup.sh]: Thank you @vuntz I was wondering how this will looks like

File: scripts/qa_crowbarsetup.sh [hunk 3]
```diff
@@ -1173,13 +1173,13 @@ function create_repos_yml()
...
         SLES12-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12:/x86_64/update/ \
         SLE12-HA-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12:/x86_64/update/ \
-        SUSE-Enterprise-Storage-2-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/Storage:/2:/x86_64/update/ \
         >> $tmp_yml
 
     create_repos_yml_for_platform "suse-12.1" "x86_64" "$tftpboot_repos12sp1_dir" \
         SLES12-SP1-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12-SP1:/x86_64/update/ \
         SLE12-SP1-HA-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12-SP1:/x86_64/update/ \
         SUSE-OpenStack-Cloud-6-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/OpenStack-Cloud:/6:/x86_64/update/ \
+        SUSE-Enterprise-Storage-2.1-Updates-test=http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/Storage:/2.1:/x86_64/update/ \
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: repo does not exist yet

Comment (mjura) [inline scripts/qa_crowbarsetup.sh]: No worries, it is already commented out in https://github.com/SUSE-Cloud/automation/pull/656/files#diff-c202af9895fd7be9224b15a93a00b0ffR536
This repo will not be used

**Comment by mjura** [comment]
Ready to merge

**Comment by mjura** [comment]
Rebased

**Comment by dirkmueller** [comment]
+1

**Comment by vuntz** [comment]
Hrm, this breaks ceph testing for M1 to M7 :/

**Pull request merged**